Config/SQLQuery_insert test values.sql

```diff
@@ -0,0 +1,572 @@
+--test1@test.com
+--Qwerty1!
+
+--test2@test.com
+--Qwerty2@
+
+--test3@test.com
+--Qwerty3%
+
+--test4@test.com
+--Qwerty4$
+
+--USE [OosBackend]
+--GO
+
+--==================== USERS ================================
+INSERT INTO [dbo].[AspNetUsers]
+           ([Id]
+           ,[CreatingTime]
+           ,[LastLogin]
+           ,[LastName]
+           ,[MiddleName]
+           ,[FirstName]
+           ,[Role]
+           ,[UserName]
+           ,[NormalizedUserName]
+           ,[Email]
+           ,[NormalizedEmail]
+           ,[EmailConfirmed]
+           ,[PasswordHash]
+           ,[SecurityStamp]
+           ,[ConcurrencyStamp]
+           ,[PhoneNumber]
+           ,[PhoneNumberConfirmed]
+           ,[TwoFactorEnabled]
+           ,[LockoutEnd]
+           ,[LockoutEnabled]
+           ,[AccessFailedCount]
+           ,[IsRegistered])
+     VALUES
+           ('16575ce5-38e3-4ae7-b991-4508ed488369' --Id
+           ,'2021-06-04 10:06:32.9282504'
+           ,'0001-01-01 00:00:00.0000000'
+           ,'������������' --last name
+           ,'������' --middle name
+           ,'����' --first name
```

Comment:
if you have time - can you check if your SQL file is saved in utf-8 encoding? because it looks like some windows 1250 :)

Reply:
Done